testapp/build.gradle

```diff
@@ -1,14 +1,14 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    compileSdkVersion 21
+    buildToolsVersion "21.1.2"
     useLibrary 'org.apache.http.legacy'
 
     defaultConfig {
         applicationId "com.microsoft.aad.testapp"
         minSdkVersion 18
-        targetSdkVersion 23
+        targetSdkVersion 21
```

Comment:
we should keep all the sample apps target as 21 unless we're ready to update to 23.